### ISSUE TITLE
Fedora KDE: Could not change permissions for "file.jpg"

### ISSUE THREAD

**Issue by matthieugras** [rfjakob/gocryptfs#271]
I'm getting this error message when I'm trying to copy some files in Dolphin (GUI file explorer) to the the crypted folder. If I copy too many files this opens so many windows that my system freezes.

**Comment by rfjakob** [comment]
Hi! I cannot reproduce this problem here. Do you mount from a network drive or something?

![screenshot from 2018-11-04 14-56-50](https://user-images.githubusercontent.com/286847/47965020-e7b9a700-e041-11e8-93b1-0528c295e50c.png)

**Comment by xelra** [comment]
I've just run into a similar problem on Arch Linux. The system is unable to set attributes and hence chmod doesn't work.

The issue is already tracked [there](https://bugs.archlinux.org/task/60967), but seeing that this issue also exists in Fedora, I believe this could be an upstream issue.

I don't know anything about the Fedora package, but I in the Arch package, this issue occurred when the build source changed from cloning the repo with tag 1.6 to getting gocryptfs_v1.6_src-deps.tar.gz.

The former also only depended on go-2, while the latter depends on a hell of a lot Haskell packages.

**Comment by rfjakob** [comment]
I agree, sounds like a problem in gocryptfs. Can I download the arch linux gocryptfs binary somewhere?

**Comment by lechner** [comment]
FWIW, I haven't noticed any problems in Debian. We built from [gocryptfs_v1.6_src.tar.gz](https://github.com/rfjakob/gocryptfs/releases/download/v1.6/gocryptfs_v1.6_src.tar.gz) (no deps).

For identification purposes, I attached your relevant upstream signature [gocryptfs_v1.6_src.tar.gz.asc.gz](https://github.com/rfjakob/gocryptfs/files/2671699/gocryptfs_v1.6_src.tar.gz.asc.gz) (Github required a zipped upload). You can also find hashes [here](http://cdn-fastly.deb.debian.org/debian/pool/main/g/gocryptfs/gocryptfs_1.6-1.dsc).

**Comment by lechner** [comment]
You can find version information for our build dependencies in [this `i386` build log](https://buildd.debian.org/status/fetch.php?pkg=gocryptfs&arch=i386&ver=1.6-1&stamp=1541205091&raw=0) (although I and most other users are probably on `amd64`)

**Comment by rfjakob** [comment]
I can reproduce the problem with gocryptfs_v1.6_src-deps.tar.gz .

**Comment by rfjakob** [comment]
So I think you need this commit: https://github.com/rfjakob/gocryptfs/commit/658cc4aebba2f1e328911cf6e2f9f6c9d1084a6c . I have released gocryptfs v1.6.1 that includes this commit: https://github.com/rfjakob/gocryptfs/releases/tag/v1.6.1

This only affects users who are on Go 1.11 or higher. Debian has Go 1.10, so I guess that's the difference.

**Comment by rfjakob** [comment]
@xelra @matthieugras does gocryptfs v1.6.1 fix the issue?

**Comment by matthieugras** [comment]
On Wednesday, December 12, 2018 9:45:23 PM CET rfjakob wrote:
> @xelra @matthieugras does gocryptfs v1.6.1 fix the issue?
I'm sorry, I can't test it because I'm not using Fedora anymore

**Comment by xelra** [comment]
@rfjakob I can confirm that the issue is fixed with v1.6.1. I was also seeing the EOPNOTSUPP errors, so I think the fix was spot on.

Strange that the error only showed on a certain build source.

IMO this issue can be closed.
